### ISSUE TITLE
Failed to activate the git-links package

### ISSUE THREAD

**Issue by kevincantu** [calebmeyer/git-links#3]
Oho, this is interesting.  Your last set of commits looked good when I took a glance at them a few days ago, but now I've updated the package and went and tried to run a git-links command (via CTRL+SHIFT+P dialog) and got a big red popup about a failure to activate.  Very close! 

**Atom**: 1.18.0 x64
**Electron**: 1.3.15
**OS**: Microsoft Windows 7 Professional 
**Thrown From**: [git-links](https://github.com/calebmeyer/git-links) package 0.3.0

### Stack Trace

Failed to activate the git-links package

```
At unmatched OUTDENT in file:///C:/Users/kevin/.atom/packages/git-links/lib/git-links.coffee:89:32

SyntaxError: unmatched OUTDENT
  at /packages/git-links/lib/git-links.coffee:89:32
```

### Commands

```
  2x -5:10.6.0 vim-mode-plus:repeat-search (input.hidden-input)
     -5:07.1.0 vim-mode-plus:put-after (input.hidden-input)
     -5:06.5.0 vim-mode-plus:move-up (input.hidden-input)
 22x -5:05.4.0 vim-mode-plus:move-right (input.hidden-input)
  2x -5:03.6.0 vim-mode-plus:delete (input.hidden-input)
  5x -5:02.2.0 vim-mode-plus:repeat-search (input.hidden-input)
     -4:59.6.0 core:save (input.hidden-input)
     -4:58.4.0 vim-mode-plus:repeat-search (input.hidden-input)
     -4:57.5.0 vim-mode-plus:move-to-last-character-of-line (input.hidden-input)
     -4:57.2.0 vim-mode-plus:insert-after (input.hidden-input)
     -4:56.8.0 vim-mode-plus:activate-normal-mode (input.hidden-input)
  4x -4:56.1.0 vim-mode-plus:repeat-search (input.hidden-input)
     -4:53.9.0 core:save (input.hidden-input)
     -0:28.2.0 intentions:highlight (input.hidden-input)
     -0:27.9.0 command-palette:toggle (input.hidden-input)
     -0:21.3.0 git-links:copy-absolute-link-for-current-line (input.hidden-input)
```

### Non-Core Packages

```
atom-fuzzy-grep 0.16.0 
busy-signal 1.4.3 
git-links 0.3.0 
go-debug 1.4.3 
go-plus 5.5.5 
go-signature-statusbar 1.2.0 
hyperclick 0.0.40 
intentions 1.1.2 
language-batchfile 0.6.3 
language-mathematica 0.3.2 
linter 2.2.0 
linter-mathematica 0.6.0 
linter-ui-default 1.6.1 
rainbow-theme 1.7.0 
sll-tools 0.0.1 
summerfruit-theme 1.9.0 
vim-mode-plus 0.93.0 
```

**Comment by kevincantu** [comment]
This file looks totally fine to my eyes, but I don't know what an OUTDENT is so um CoffeeScript is drunk?
https://github.com/calebmeyer/git-links/blob/master/lib/git-links.coffee#L89

**Comment by calebmeyer** [comment]
Turns out the new function I added was all kinds of messed up. Need two closing parens and an indent. Fixed in 3b28be7, published in 0.3.1

Please let me know if it works again.

**Comment by kevincantu** [comment]
It's working well!! :smiley_cat:
